packages/core/lib/patchers/aws3_p.ts

```diff
@@ -18,21 +18,28 @@ var contextUtils = require('../context_utils');
 
 var logger = require('../logger');
```

Comment:
Use let/const instead of var, as this is a TypeScript file.
```suggestion
const logger = require('../logger');
```